src_c/surface.c

```diff
@@ -1699,6 +1700,7 @@ surf_convert(PyObject *self, PyObject *args)
     Uint32 colorkey;
     Uint8 key_r, key_g, key_b, key_a = 255;
     int has_colorkey = SDL_FALSE;
+    key_b = 0;
```

Comment:
key_a has been init'ed with 255... whereas key_b = 0. Not sure what the dev wants.